src/cljs/rems/catalogue.cljs

```diff
@@ -139,7 +139,7 @@
                               :col-span #(if (:category/id %) 2 1)}
                              {:key :commands
                               :content #(when-not (:category/id %)
-                                          [:div.commands.w-100
+                                          [:div.commands.w-100.justify-content-end
```

Comment:
Default is already the end?

Reply:
Default for a table is actually start, so for the cart and catalogue we must override it.